js/assessments/calculateFleschReading.js

```diff
@@ -0,0 +1,99 @@
+var calculateFleschReading = require( "../analyses/calculateFleschReading.js" );
+var AssessmentResult = require( "../values/AssessmentResult.js" );
+var inRange = require( "lodash/number/inRange" );
+
+/**
+ * Calculates the assessment result based on the fleschReadingScore
+ * @param {int} fleschReadingScore The score from the fleschReadingtest
+ * @param {object} i18n The i18n-object used for parsing translations
+ * @returns {object} object with score, resultText and note
+ */
+var calculateFleschReadingResult = function( fleschReadingScore, i18n ) {
+	if ( fleschReadingScore > 90 ) {
+		return {
+			score: 9,
+			resultText: i18n.dgettext( "js-text-analysis", "very easy" ),
```

Comment:
A translator comment isn't really necessary, since we don't replace anything in this string.